gandi/cli/tests/commands/test_contact.py

```diff
@@ -152,5 +152,7 @@ def test_create_apikey_ok(self):
 Will save your apikey into the config file.""")
         self.assertEqual(result.exit_code, 0)
 
-        self.assertEqual(GandiModule._conffiles,
```

Comment:
I never tested with a local configuration file, as I use tox or travis to run tests, but your point is valid indeed, if it can avoid a failed test you can keep your changes, sorry to answer late as I see you already have reverted the change